test/services/licences/determine-supplementary-billing-years.service.test.js.js

```diff
@@ -0,0 +1,121 @@
+'use strict'
+
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+const Sinon = require('sinon')
+
+const { describe, it, beforeEach, afterEach } = exports.lab = Lab.script()
+const { expect } = Code
+
+// Thing under test
+const DetermineSupplementaryBillingYearsService = require('../../../app/services/licences/determine-supplementary-billing-years.service.js')
+
+describe('Supplementary Billing Years Service', () => {
+  let startDate
+  let endDate
+  let clock
+  let testDate
+
+  afterEach(() => {
+    Sinon.restore()
+  })
+
+  describe('when given a start date beginning before April', () => {
+    beforeEach(() => {
+      startDate = new Date('2023-03-20')
+    })
+
+    afterEach(() => {
+      clock.restore()
+    })
+
+    describe('and no end date', () => {
+      beforeEach(() => {
+        testDate = new Date('2024-03-31')
+        clock = Sinon.useFakeTimers(testDate)
+
+        endDate = null
+      })
```

Comment:
We repeat this same section in the next section of tests. So, in effect it applies to _all_ unit tests in this file.

On that basis, I'd suggest moving this to a top-level `before()`, and the `clock.restore()` along with the `Sinon.restore()` into a top-level `after()`.